src/app/containers/AssignmentCreateForm.tsx

```diff
@@ -7,24 +7,31 @@ import {
 } from '../api/index';
 import { createAssignment } from '../modules/assignments/actions';
 import { default as FormSubmitButton, SubmitButtonProps } from '../components/FormElements/SubmitButton'
+import { connect } from 'react-redux';
+import { RootState } from '../store';
 
 
 // wrapping generic assignment form in redux-form
 const formConfig: ConfigProps<any, AssignmentFormProps> = { 
     form: 'CreateAssignment',
     onSubmit: (values:{assignment: SheriffAssignment}, dispatch, props) =>{
-        const { workSectionId } = props;
-
         let newAssignment = Object.assign({}, values.assignment,
-            {requiredAbilities:SheriffAbility.All, sheriffIds:[], workSectionId}, 
+            {requiredAbilities:SheriffAbility.All, sheriffIds:[]}, 
         );
         dispatch(createAssignment(newAssignment));
     } 
 };
 
+const mapStateToProps = (state: RootState, props: AssignmentFormProps) => {
+    return {
+        initialValues: {assignment: {workSectionId: props.workSectionId}},
```

Comment:
Lots of times when writing objects I'll put them in nested looking structure just for readability sake:
```ts
{
  initialValues: {
    assignment:{
        workSectionId: props.worksectionId
    }
  },
  isDefaultTemplate: false
}
```